packages/core/.storybook/preview.js

```diff
@@ -38,6 +39,12 @@ const Style = ({ children, styles }) => {
   return children;
 };
 
+let isDev = CONFIG_TYPE === 'DEVELOPMENT'; //  process.env?.NODE_ENV === 'development';
```

Comment:
Don't think you need to use `let` here. Also, where is `CONFIG_TYPE` coming from? 🤔

Reply:
We have a `isProduction` variable that could possibly be used from `packages/ibm-products/src/global/js/utils/pconsole.js`.

Reply:
As far as I can tell Storybook does not pass the NODE_ENV parameter

Reply:
CONFIG_TYPE is added by storybook I believe

https://github.com/search?q=repo%3Astorybookjs%2Fstorybook%20config_type&type=code